__tests__/setup.ts

```diff
@@ -26,6 +26,7 @@ jest.mock('../src/remoteConfig', () => ({
       vordrIps: ['192.0.2.0/24', '198.51.100.0/24', '203.0.113.0/24'],
       ignoredWorkEmailDomains: ['igored.com', 'ignored.org'],
       rateLimitReputationThreshold: 1,
+      pricingIds: { pricingGift: 'yearly' },
```

Comment:
PlusPriceType.Yearly :) but not urgent

Reply:
Tried to use `SubscriptionCycles` but didn't work due to load time, reverted and maybe add later on